app/assets/stylesheets/layout_components/card.scss

```diff
@@ -0,0 +1,41 @@
+.card {
+  background: $white;
+  padding: $base-padding 0;
+  text-align: center;
+  margin-bottom: $base-margin;
+
+  &__title {
+    font-size: $base-font-size*1.45;
+    padding: 7px;
+    font-weight: bold;
+    text-align: center;
+
+    @include breakpoint(small only) {
+      font-size: $base-font-size*1.25;
+    }
+  }
+
+  &--blue-bg {
+    @extend .card__title;
+    background: $brand-blue-light;
+    color: $white;
+    position: relative;
+  }
+
+  &__image {
+    margin-bottom: $base-margin;
+  }
+
+  &__content-bg {
+    background: $ash;
+    border-top: 2px solid $stone;
+    border-bottom: 2px solid $stone;
+    padding: $base-padding 0;
+    margin-bottom: $base-margin;
+  }
+
+  &--content {
+    padding: 0 $base-padding;
+    margin-bottom: 0;
```

Comment:
i think instead of making the margin-bottom 0 here, it would be better to change the padding on the content-bg to not have bottom padding - otherwise if there were 2 paragraphs, they wouldn't have margin between them

Reply:
I only put margin bottom to zero here to override the bottom margin on the p tag. I also thought it was doubtful we'd ever allow two paragraphs in this small content area? So I thought this was a case where we would always want to over ride the bottom margin from the p tag.

Reply:
knowing rewards, I'm sure someday we will be forced to break that rule :) So i would say change it so it works in either case.

Reply:
we don't need either anymore after some of these other changes i've done.